docs/guide/reporters.md

```diff
@@ -151,7 +172,7 @@ Example output using basic reporter:
 
 ### Verbose Reporter
 
-Follows the same hierarchical structure as the `default` reporter, but does not collapse sub-trees for passed test suites. The final terminal output displays all tests that have run, including those that have passed.
+Verbose reporter is same as `default` reporter, but it also displays each individual test after the suite has finished. Similar to `default` reporter, you can disable the summary by configuring the reporter.
```

Comment:
hm, it is quite useful to show currently running tests especially when they timeout

Reply:
Yep I agree. However showing all tests like current `verbose` reporter does causes a lot of flickering as there is simply too much data that changes too fast.

How about we show names of running tests that have been running longer than `slowTestThreshold`?

Then we could see slow tests like:

<img width="400" src="https://github.com/user-attachments/assets/104b7d86-dd1d-4af7-8980-917214d5db54" />

And if there are other fast running tests, they will not expand:

<img width="400" src="https://github.com/user-attachments/assets/e1bcc429-8141-4a5b-b743-6b826055fc8d" />

Reply:
sure, seems like a good middle ground 👌
but don't forget they can run in parallel, so maybe we limit the number of tests shown?
and also beforeAll/beforeEach can take time (we previously were showing it)

Reply:
Now tracking slow hooks too. The `onTaskUpdate` is tricky 😬 

```ts
import { afterAll, afterEach, beforeAll, beforeEach, test } from 'vitest'

beforeEach(async () => {
  await new Promise(resolve => setTimeout(resolve, 3_000))
})

beforeAll(async () => {
  await new Promise(resolve => setTimeout(resolve, 3_000))
})

afterEach(async () => {
  await new Promise(resolve => setTimeout(resolve, 3_000))
})

afterAll(async () => {
  await new Promise(resolve => setTimeout(resolve, 3_000))
})

test('fixture 1', async () => {
  await new Promise(resolve => setTimeout(resolve, 3_000))
})
```

[summary-hooks.webm](https://github.com/user-attachments/assets/6a4f1bb0-d740-4cfb-b1d0-2f726b586905)

Reply:
Oh wow, looks awesome!